File: sdkwork-backend-sdk-go/types/plus_oss_bucket_vo.go
package types

// OSS存储桶值对象
type PlusOssBucketVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	V int `json:"v"`
	UserId int `json:"userId"`
	Name string `json:"name"`
	Region string `json:"region"`
	Channel string `json:"channel"`
	ChannelConfigId int `json:"channelConfigId"`
	Description string `json:"description"`
	Status string `json:"status"`
	CreationDate string `json:"creationDate"`
	Arn string `json:"arn"`
	Endpoint string `json:"endpoint"`
	StorageClass string `json:"storageClass"`
	VersioningEnabled bool `json:"versioningEnabled"`
	EncryptionEnabled bool `json:"encryptionEnabled"`
	EncryptionType string `json:"encryptionType"`
	Acl string `json:"acl"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusOssBucketVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusOssBucketVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? V { get; set; }
        public int? UserId { get; set; }
        public string? Name { get; set; }
        public string? Region { get; set; }
        public string? Channel { get; set; }
        public int? ChannelConfigId { get; set; }
        public string? Description { get; set; }
        public string? Status { get; set; }
        public string? CreationDate { get; set; }
        public string? Arn { get; set; }
        public string? Endpoint { get; set; }
        public string? StorageClass { get; set; }
        public bool? VersioningEnabled { get; set; }
        public bool? EncryptionEnabled { get; set; }
        public string? EncryptionType { get; set; }
        public string? Acl { get; set; }
    }
}
